companion/src/simulation/telemetrysimu.cpp

```diff
@@ -59,6 +59,8 @@ TelemetrySimulator::TelemetrySimulator(QWidget * parent, SimulatorInterface * si
   connect(&timer,    &QTimer::timeout, this, &TelemetrySimulator::generateTelemetryFrame);
   connect(&logTimer, &QTimer::timeout, this, &TelemetrySimulator::onLogTimerEvent);
 
+  gpsTimer.setInterval(250);
+  connect(&gpsTimer,&QTimer::timeout,this,&TelemetrySimulator::onGpsRunLoop);
```

Comment:
For visual consistency use style of next line

Reply:
Do you mean I should delete line 61?

Reply:
Space the parameters like line 64 just for readability. Breaks the coding style of a single space after each comma but in this case it helps.

Reply:
I understand what you are asking for now. However, since line 64 is not a part of my additional code, perhaps I should add single spaces after the commas and an additional line after line 63. It would then look like this:
61
62  gpsTimer.setInterval(250);
63  connect(&gpsTimer, &QTimer::timeout, this, &TelemetrySimulator::onGpsRunLoop);
64  
65 connect(ui->Simulate,          &QCheckBox::toggled, [&](bool on) { g.currentProfile().telemSimEnabled(on);         });

Reply:
ok